index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tangerine">
+    <title>MEdicore</title>
+    <link href="./styles.css" rel="stylesheet">
+</head>
+<body>
+    <div class="section--pink">
+        <div class="menu-bar">
+            <div class="menu-bar__section menu-bar__section--text-big">
+                <div class="menu-bar__link">Medical Core</div>
+            </div>
+            <div class="menu-bar__section menu-bar__section--text-muted menu-bar__links">
+                <div class="menu-bar__link">Home</div>
+                <div class="menu-bar__link">Product</div>
+                <div class="menu-bar__link">Pricing</div>
+                <div class="menu-bar__link">Contact</div>
+            </div>
+            <div class="menu-bar__section">
+                <button class="button button--white">Login</button>
+                <button class="button button--uppercase button--blue">join us <img src="./images/Vector.png"> </button>
+            </div>
+        </div>
+        <div class="welcome">
+            <div class="welcome__content">
+                <label class="welcome__text--blue welcome__text--big-in-sm">Welcome</label>
+                <label class="welcome__text--big">A Great Place to Receive Care</label>
+                <label class="welcome__text--muted">Get your best looking smile now!</label>
+                <div class="welcome__buttons">
+                    <button class="button button--blue">Join Us</button>
+                    <button class="button button--border">Learn More</button>
+                </div>
+            </div>
+            <div class="welcome__empty"></div>
+        </div>
+    </div>
+    <div class="content">
+        <div class="content__image">
+            
+            <img class="content__image--1" src="./images/lady-doctor.png">
+        </div>
+        <div class="content__text">
+            <div>
+                <hr class="content__hr">
+                <label class="content__text--big">Meet our experts</label>
+                <div class="content__text--muted">
+                    Problems trying to resolve the conflict between 
+                    the two major realms of Classical physics:
+                    Newtonian mechanics 
+                    <div class="welcome__text--blue text--padded">
```

Comment:
According to BEM, modifiers are preceded by block or element class. This would have been something like `<div class="welcome__text welcome__text--blue ...">`

Reply:
Thanks! mate.